src/pages/Checkout/CheckoutPage.vue

```diff
@@ -489,9 +492,8 @@ export default {
 <style lang="scss">
 @import 'settings';
 
-.page-content {
-	padding: 1.625rem 0;
-
+.page-content,
+.empty-basket {
```

Comment:
various styles to apply to loader now that it's alongside of page-content rather than inside that container.